includes/class.llms.cache.helper.php

```diff
@@ -82,6 +82,9 @@ public static function invalidate_group( $group ) {
 	 * This prevents caching for the Checkout & Student Dashboard pages.
 	 *
 	 * @since 3.15.0
+	 * @since [version] Force no caching on quiz pages.
+	 *               Added `llms_no_cache` filter hook.
+	 *               Added 'no-store' to the default WordPress nocache headers.
```

Comment:
```suggestion
	 * @since [version] Force no caching on quiz pages.
	 *               Added 'no-store' to the default WordPress nocache headers.
```

Hook introduction is documented on the hook itself so this is unnecessary